lib/WTSI/NPG/HTS/PacBio/Sequel/APIClient.pm

```diff
@@ -127,7 +129,7 @@ Guoying Qi E<lt>gq1@sanger.ac.ukE<gt>
 
 =head1 COPYRIGHT AND DISCLAIMER
 
-Copyright (C) 2011, 2016 Genome Research Limited. All Rights Reserved.
+Copyright (C) 2017 Genome Research Limited. All Rights Reserved.
```

Comment:
Should be "Copyright (C)  2011,  2016,  2017" to comply with WTSI software licensing policy for boilerplate